File: implementation/scipamato/scipamato-public-web/src/test/java/ch/difty/scipamato/publ/web/resources/IcoMoonIconTypeTest.java
package ch.difty.scipamato.publ.web.resources;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class IcoMoonIconTypeTest {

    @Test
    void arrow_right() {
        assertThat(IcoMoonIconType.arrow_right.cssClassName()).isEqualTo("icon-arrow-right");
    }

    @Test
    void link() {
        assertThat(IcoMoonIconType.link.cssClassName()).isEqualTo("icon-link");
    }
}

File: public/public-web/src/test/kotlin/ch/difty/scipamato/publ/web/resources/IcoMoonIconTypeTest.kt
package ch.difty.scipamato.publ.web.resources

import org.amshove.kluent.shouldBeEqualTo
import org.junit.jupiter.api.Test

class IcoMoonIconTypeTest {

    @Test
    fun arrow_right() {
        IcoMoonIconType.arrow_right.cssClassName() shouldBeEqualTo "icon-arrow-right"
    }

    @Test
    fun link() {
        IcoMoonIconType.link.cssClassName() shouldBeEqualTo "icon-link"
    }
}
